### PR TITLE
Clean up readme and arguments of chunked prefill

### PR DESCRIPTION
**Pull request by merrymercy**

### CODE REVIEW

**Comment by zhyncs** [comment]
Consider waiting for https://github.com/sgl-project/sglang/pull/1013

**Comment by zhyncs** [comment]
> Consider waiting for #1013

ignore this one

File: python/sglang/srt/server_args.py [hunk 1]
```diff
@@ -43,10 +43,11 @@ class ServerArgs:
 
     # Memory and scheduling
     mem_fraction_static: Optional[float] = None
-    max_prefill_tokens: Optional[int] = None
     max_running_requests: Optional[int] = None
     max_num_reqs: Optional[int] = None
     max_total_tokens: Optional[int] = None
+    chunked_prefill_size: int = 8192
```
Comment (merrymercy) [inline python/sglang/srt/server_args.py]:
```suggestion
    chunked_prefill_size: int = -1
```

File: README.md [hunk 2]
```diff
@@ -139,23 +139,23 @@ print(response)
...
 python -m sglang.launch_server --model-path meta-llama/Meta-Llama-3-8B-Instruct --port 30000 --dp 2 --tp 2
 ```
-- If you see out-of-memory errors during serving, please try to reduce the memory usage of the KV cache pool by setting a smaller value of `--mem-fraction-static`. The default value is `0.9`.
+- If you see out-of-memory errors during serving, try to reduce the memory usage of the KV cache pool by setting a smaller value of `--mem-fraction-static`. The default value is `0.9`.
 ```
 python -m sglang.launch_server --model-path meta-llama/Meta-Llama-3-8B-Instruct --port 30000 --mem-fraction-static 0.7
 ```
-- If you see out-of-memory errors during prefill for long prompts on a model that supports long context, consider using chunked prefill.
+- See [hyperparameter_tuning.md](docs/en/hyperparameter_tuning.md) on tuning hyperparameters for better performance.
+- If you see out-of-memory errors during prefill for long prompts, try to set a smaller prefill chunk size.
```
Comment (merrymercy) [inline README.md]:
```suggestion
- If you see out-of-memory errors during prefill for long prompts, try to set a smaller chunked prefill size.
```

**Pull request merged**